client/packages/requisitions/src/RnRForms/DetailView/ContentArea.tsx

```diff
@@ -84,7 +84,7 @@ export const ContentArea = ({
           '.first-column': {
             left: 0,
             position: '-webkit-sticky',
-            width: 80,
+            minWidth: 80,
```

Comment:
code column was still jumping around on scroll -

Reply:
I had to give `second-column` a `left: 82` to prevent the border disappearing on scroll though. 
Is it ok for you?

This is how my screen looks at `80`

https://github.com/user-attachments/assets/55abc4f5-5204-4e2c-a406-c2c11176411f

Reply:
Oh, I think some of your codes are longer than any I have - maybe we need to go a little bigger, and add an overflow?